main.js

```diff
@@ -0,0 +1,51 @@
+// Create and show categories
+
+let categoriesConteiner = document.getElementById("categories");
+let searchProduct = document.getElementById("product-list");
+let productSelectElement = document.getElementById("search-product");
+let fieldForProductSelect = document.getElementById("total");
+
+document.body.onload = renderCategories(categories, categoriesConteiner);
+
+createProductElementOption(products);
+
+// find and show selected product
+
+productSelectElement.addEventListener("input", (event) => {
+  const productSelectElement = products.filter(
+    (search) => search.name === event.target.value
+  );
+  const productElement = buildProductElement(productSelectElement);
+  fieldForProductSelect.appendChild(productElement);
```

Comment:
Change the name of the variable:
```
const foundProducts = products.filter(
    (search) => search.name === event.target.value
  );
```
Now, when you have a list of products `foundProducts` you can use a loop for walking through all of them and rendering them (the same approach you used in the `renderCategories`).

Reply:
done with follow code: 
 productSelectElement.addEventListener('input', (event) => {
  const foundProduct = products.find((product) => product.name === event.target.value);
  if (foundProduct) {
    const productElement = buildProductElement(foundProduct);
    fieldForProductSelect.appendChild(productElement);
  }
});